scripts/science_archive/ingest_sample_data.py

```diff
@@ -0,0 +1,217 @@
+# Import the ingester only after the needed environment variables have been set.
+################################################################################
+from ocs_ingester import ingester
+
+
+################################################################################
+# Create a temporary directory for working with our fits file.
+# The goal is to run a loop that modifies our sample data's metadata to match
+# a completed observation, and then upload that file. 
+################################################################################
+
+# Make a temporary directory
+tmp_fits_dir =  f'/example_data/tmp_fits'
+try:
+    os.makedirs(tmp_fits_dir)
+except OSError:
+    print('Could not create the directory ', tmp_fits_dir)
+    pass
+
+# copy the fits file so that we can modify it without changing the original
+file = 'ogg0m406-kb27-20210720-0305-s91.fits.fz'
+original_fits_path = f'/example_data/{file}'
+copied_fits_path = f'/example_data/tmp_fits/{file}'
+shutil.copyfile(original_fits_path, copied_fits_path)
+filename_counter = -1
+
+################################################################################
+# Helpers to get completed observations so we can make matching data files
+################################################################################
+def get_completed_observations():
+    observations_endpoint = f"{os.getenv('OBSERVATION_PORTAL_BASE_URL')}/observations"
+    query_params = {
+        "state": "COMPLETED",
+        "user": "test_user",
+    }
+    observations_endpoint += "?" + urllib.parse.urlencode(query_params)
+    obs_auth_header = { "Authorization": f"Token {observations_token}" }
+
+    observations = requests.get(observations_endpoint, headers=obs_auth_header).json()
+    pprint(len(observations["results"]))
+    return observations["results"]
+
+def header_vals_from_observation(obs):
+    return {
+        #"RLEVEL": "",
+        "DAY_OBS": obs['end'].split('T')[0],
+        "DATE_OBS": obs['end'],
+        "PROPID": obs['proposal'],
+        "INSTRUME": obs['request']['configurations'][0]['instrument_name'],
+        "OBJECT": obs['request']['configurations'][0]['target']['name'],
+        "SITEID": obs['site'],
+        "TELID": obs['telescope'],
+        "EXPTIME": obs['request']['configurations'][0]['instrument_configs'][0]['exposure_time'],
+        "FILTER": obs['request']['configurations'][0]['instrument_configs'][0]['optical_elements']['filter'],
+        "OBSTYPE": obs['request']['configurations'][0]['type'],
+        "BLKUID": obs['id'],
+        "REQNUM": obs['request']['configurations'][0]['id'],
+        "TELID": obs['telescope'],
+    }
+
+def filename_from_observation_header_vals(header_vals):
+    filename = f"{header_vals['SITEID']}{header_vals['TELID']}"
+    filename += f"-{header_vals['INSTRUME']}-{header_vals['DAY_OBS'].replace('-', '')}"
+    filename += f"-{str(filename_counter).zfill(4)}-e91.fits.fz"
```

Comment:
Similar comment to above - this assumes an EXPOSE type observation with reduction level 91. Not a big deal though.